src/commands/config.rs

```diff
@@ -91,16 +91,37 @@ pub struct Config {
     #[structopt(long = "--data-dir", env = "INFLUXDB_IOX_DB_DIR")]
     pub database_directory: Option<PathBuf>,
 
+    #[structopt(
+        long = "--object-store",
+        env = "INFLUXDB_IOX_OBJECT_STORE",
+        possible_values = &ObjectStore::variants(),
+        case_insensitive = true,
+        long_help = r#"Which object storage to use. If not specified, defaults to memory.
+
+Possible values (case insensitive):
+
+* memory (default): Effectively no object persistence.
+* file: Stores objects in the local filesystem. Must also set `--database_directory`.
```

Comment:
```suggestion
* file: Stores objects in the local filesystem. Must also set `--data-dir`.
```

`--database-directory` wasn't an option when I tried this out locally :)

Reply:
oooops, great catch! I read the Rust field name instead of the structopt flag 🤦‍♀️